nupic/research/frameworks/dynamic_sparse/common/browser.py

```diff
@@ -163,7 +222,17 @@ def _get_value(
 
         # add all data for one metric - required to plot
         for m in full_metrics:
-            stats[m + "_all"].append(progress[e][m])
+
```

Comment:
Can't really tell what is this for - I trust you've tested the browser class after the changes, and it is working same as usual?

Reply:
I've only made changes that add functionality and my usage locally confirms that nothing was broken in the process. 

In this logic, I simply wanted a way to add values to `stats` that didn't need any preprocessing - that is, their raw values.